common/js/src/popup-window/popup-opener.js

```diff
@@ -32,6 +32,22 @@ goog.require('goog.object');
 
 goog.scope(function() {
 
+const GSC = GoogleSmartCard;
+
+/**
+ * The window options to be used for the resulting window. 
```

Comment:
Please document which options are supported in which mode (as I see that some of them are ignored in the Extension mode, presumably intentionally).